h/models/annotation.py

```diff
@@ -221,5 +221,11 @@ def authority(self):
             return None
         return split_user(self.userid)["domain"]
 
+    @property
+    def is_hidden(self):
+        """Check if this annotation id is hidden."""
+
+        return self.moderation is not None
```

Comment:
Not sure a service is appropriate for this. I think we might be reaching for services as a first resort rather than thinking about whether they aid understanding or obscure what's going on.

Reply:
Working on the attributes of the annotation itself, not making any extra queries, I'd say it fits the model 100% here.